Documentation/specs/runtime-configuration-file.md

```diff
@@ -200,6 +199,24 @@ In the framework-dependent deployment model, the `*.runtimeConfig.json` file wil
 
 This data is used to locate the shared framework folder. The exact mechanics of which version are selected are defined elsewhere, but in general, it locates the shared runtime in the `shared` folder located beside it by using the relative path `shared/[runtimeOptions.framework.name]/[runtimeOptions.framework.version]`. Once it has applied any version roll-forward logic and come to a final path to the shared framework, it locates the `[runtimeOptions.framework.name].deps.json` file within that folder and loads it **first**.
 
+Note, starting with 3.0, the "framework" section is optional and a new "frameworks" section supports multiple frameworks:
+```json
+{
+    "runtimeOptions": {
+        "frameworks": [
+            {
+                "name": "Microsoft.AspNetCore.All",
+                "version": "3.0.0"
+            },
+            {
+                "name": "Microsoft.Forms",
+                "version": "3.0.0"
+            }
+        ]
+    }
+}
+```
+
```

Comment:
What is the semantics if both `framework` and `frameworks` sections are defined? We should state it here.

Reply:
Sure. I added some information about that